book/src/debugging.md

```diff
@@ -0,0 +1,127 @@
+# Debugging
+
+Gain deeper insights into your code's behavior with this debugging section. Here you can discover techniques and tools to explore and understand your project's internals.
```

Comment:
I'd reword this, I think this is about debugging Logos's lexer behavior and not my project internals.